{"name": "test_access_points_connectivity", "status": "passed", "description": "Test case to verify Access Points Connectivity", "attachments": [{"name": "Access Point Connectivity Failed", "source": "32419a70-2e11-436d-92c6-7ab9f15dcb05-attachment.attach"}, {"name": "Manager status is Active", "source": "ece31418-22d1-45f3-87ab-163abec04678-attachment.attach"}, {"name": "log", "source": "bbe902bc-5dd4-48fc-93bd-0fcde998a171-attachment.txt", "type": "text/plain"}], "start": 1626568225129, "stop": 1626568225130, "uuid": "6926d59d-be09-4a08-96f1-05e77da46860", "historyId": "9c9c1794599fbd8dfa1d00f937bef4d8", "testCaseId": "b1de51dd692b456a6a233ecea89ec452", "fullName": "test_connectivity.TestResources#test_access_points_connectivity", "labels": [{"name": "tag", "value": "test_access_points_connectivity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_test_run')"}, {"name": "tag", "value": "sanity_55"}, {"name": "tag", "value": "test_resources"}, {"name": "tag", "value": "sanity"}, {"name": "suite", "value": "test_connectivity"}, {"name": "subSuite", "value": "TestResources"}, {"name": "host", "value": "testing-83-mddqm"}, {"name": "thread", "value": "6-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_connectivity"}], "links": [{"type": "test_case", "name": "test_access_points_connectivity"}, {"type": "test_case", "name": "Test Resources"}]}